src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/legalrepresentative/LegalRepresentativeEmailNotificationPersonalisation.java

```diff
@@ -1,28 +1,41 @@
-
-import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.CHANGE_ORGANISATION_REQUEST_FIELD;
-import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.LEGAL_REPRESENTATIVE_EMAIL_ADDRESS;
-
-import java.util.Collections;
-import java.util.Set;
-import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
-import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.field.ChangeOrganisationRequest;
-import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.EmailNotificationPersonalisation;
-
-public interface LegalRepresentativeEmailNotificationPersonalisation extends EmailNotificationPersonalisation {
-
-    @Override
-    default Set<String> getRecipientsList(AsylumCase asylumCase) {
-
-        if (asylumCase.read(CHANGE_ORGANISATION_REQUEST_FIELD, ChangeOrganisationRequest.class)
-            .map(it -> it.getCaseRoleId() == null)
-            .orElse(false)) {
-
-            return Collections.emptySet();
-        } else {
-            return Collections.singleton(asylumCase
-                .read(LEGAL_REPRESENTATIVE_EMAIL_ADDRESS, String.class)
-                .orElseThrow(() -> new IllegalStateException("legalRepresentativeEmailAddress is not present")));
-        }
-    }
-}
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.legalrepresentative;
+
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.CHANGE_ORGANISATION_REQUEST_FIELD;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.JOURNEY_TYPE;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.LEGAL_REPRESENTATIVE_EMAIL_ADDRESS;
+
+import java.util.Collections;
+import java.util.Objects;
+import java.util.Set;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.JourneyType;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.field.ChangeOrganisationRequest;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.EmailNotificationPersonalisation;
+
+public interface LegalRepresentativeEmailNotificationPersonalisation extends EmailNotificationPersonalisation {
+
+    @Override
+    default Set<String> getRecipientsList(AsylumCase asylumCase) {
+
+        if (asylumCase.read(JOURNEY_TYPE, JourneyType.class)
+            .map(type -> Objects.equals(type.getValue(), JourneyType.AIP.getValue()))
+            .orElse(false)) {
+
+            return Collections.emptySet();
+        }
+
+        if (asylumCase.read(CHANGE_ORGANISATION_REQUEST_FIELD, ChangeOrganisationRequest.class)
+            .map(it -> it.getCaseRoleId() == null)
+            .orElse(false)) {
+
+
+
+            return Collections.emptySet();
+        } else {

```

Comment:
The `else` can be omitted here.

The subsequent lines will run anyway, if you don't enter one of the `if` statements. Removing the `else` will help redability.